src/capi/azext_capi/custom.py

```diff
@@ -597,7 +597,28 @@ def install_cni(cmd, cluster_name, workload_cfg, windows, args):
             cidr1 = retry_shell_command(command, attempts, delay)
         except subprocess.CalledProcessError:
             pass  # This is a best-effort configuration, so don't fail if we can't find the CIDR.
+    return cidr0, cidr1
+
+
+def install_cloud_provider(cmd, cluster_name, workload_cfg, cidr0, cidr1):
```

Comment:
What is cidr0 and cidr1? I'm not too familiar with the terms and it may help to add a comment explaining the parameters.

Reply:
[CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) == classless inter-domain routing, here meaning basically "What network address ranges are available to this cluster?" Definitely should add a comment; I'll do that.

Reply:
Added some comments about CIDRs and also an explanation for the "incompatible flags" warning.

Reply:
Ah I was confused about the difference/meaning of cidr0 vs cidr1 but the comment is helpful. Thanks!

Reply:
Oh if you were curious about the variable naming, it's just "first CIDR" and "second CIDR" but zero-based, I think this was copied directly from some CAPZ shell script of yore.